### ISSUE TITLE
Parse error with backslash before indented block

### ISSUE THREAD

**Issue by tusharsadhwani** [microsoft/pyright#7799]
**Describe the bug**

The following code runs fine on Python 3.10+, but pyright fails to parse it.

Spotted from https://github.com/astral-sh/ruff/issues/10099

**Code or Screenshots**

```python
class Plotter:
\
    pass
```

**VS Code extension or command-line**
```console
$ pyright foo.py
/Users/tusharsadhwani/code/black/foo.py
  /Users/tusharsadhwani/code/black/foo.py:3:5 - error: Expected indented block
1 error, 0 warnings, 0 informations 
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.361, which I just published.
